.github/workflows/release.yml

```
@@ -21,6 +21,8 @@ jobs:
           go-version: stable
       # More assembly might be required: Docker logins, GPG, etc.
       # It all depends on your needs.
+      - run: if [ "$(git describe --tags --exact-match 2> /dev/null)" = "" ]; then git tag $(git describe --tags --abbrev=0 | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g'); fi
+        # tagがなければ、一つ前のtagからパッチバージョンを1足して新しくtagをつける
```

```suggestion
      # tagがなければ、一つ前のtagからパッチバージョンを1足して新しくtagをつける
      - run: if [ "$(git describe --tags --exact-match 2> /dev/null)" = "" ]; then git tag $(git describe --tags --abbrev=0 | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g'); fi
      # More assembly might be required: Docker logins, GPG, etc.
      # It all depends on your needs.
```

コメントの位置関係はこういう感じが適切？

---

入れた